tests/testcases/030_check-network.yml

```diff
@@ -12,16 +12,17 @@
       bin_dir: "/usr/local/bin"
     when: not ansible_os_family in ["CoreOS", "Container Linux by CoreOS"]
 
-
-  - name: Wait for pods to be ready
-    shell: "{{bin_dir}}/kubectl get pods -n test"
-    register: pods
+  - name: Check that all pods are running and ready
+    shell: "{{bin_dir}}/kubectl get pods --all-namespaces --no-headers -o yaml"
```

Comment:
you should not get all namespaces here, but only the `-n test` test namespace

Reply:
Good catch, fixed in latest commit